dps-gradle-spring-boot-suppressions.xml

```diff
@@ -1,826 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
doesn't need to be checked in as dps gradle plugin will create each time now instead